thentos-core/src/Thentos/Backend/Core.hs

```diff
@@ -133,6 +133,12 @@ thentosErrorInfo other e = f e
         (Nothing, err400, "thentos session not found")
     f NoSuchServiceSession =
         (Nothing, err400, "service session not found")
+    f NoSuchPersona =
+        (Nothing, err400, "persona not found")
+    f NoSuchContext =
+        (Nothing, err400, "context not found")
+    f MultiplePersonasPerContext =
+        (Nothing, err400, "Cannot register multiple personas by a user for the same context")
```

Comment:
All other error messages start with a small letter.  And I would mention sybil attacks here:

"sybil attack prevention: cannot register multiple personas of the same user for the same context"